packages/front-end/components/Experiment/NewExperimentForm.tsx

```diff
@@ -252,6 +255,13 @@ const NewExperimentForm: FC<NewExperimentFormProps> = ({
     >
       <Page display="Basic Info">
         {msg && <div className="alert alert-info">{msg}</div>}
+
+        {currentProjectIsDemo && (
+          <div className="alert alert-warning">
+            You are creating an experiment under the demo datasource project.
```

Comment:
Should we add "This experiment will be deleted when the demo datasource project is deleted."

To here and the other similar alerts?

Reply:
Change made here 667afcdbf13a32e1f5d4c329a9cc7a8c3e7d5816